crowbar_framework/app/models/service_object.rb

```diff
@@ -1460,11 +1460,6 @@ def proposal_dependencies(role)
     []
   end
 
-  def skip_unchanged_node?(node_name, old_role, role)
-    # By default dont skip anything
-    false
-  end
```

Comment:
I would actually not change this method: we should keep it that way so that it gets overridden in each barclamp, because each barclamp may have some additional tests to decide on whether to skip or not. My request was really to put the helpers here (like you did) so they get shared.